api/libproject/devstart.rb

```diff
@@ -1327,3 +1327,26 @@ def setup_cloud_project(cmd_name, *args)
   :description => "Initializes resources within a cloud project that has already been created",
   :fn => lambda { |*args| setup_cloud_project("setup-cloud-project", *args) }
 })
+
+def start_api_and_incremental_build(cmd_name, args)
+  ensure_docker cmd_name, args
+  common = Common.new
+  begin
+    # appengineStart must be run with the Gradle daemon or it will stop outputting logs as soon as
+    # the application has finished starting.
+    common.run_inline %W{gradle --daemon appengineStart}
+    common.run_inline "tail -f -n 0 /w/api/build/dev-appserver-out/dev_appserver.out &"
```

Comment:
So the tail is running in the background? How do you get it to stop?

I would rather either not do the tail (and document where to find the logs -- see above), or do the tail and block on it.

Either way, we should document how to shut things down when you start them this way... is there a target for that?

Reply:
Exactly.